APITesters/SwiftAPITester/SwiftAPITester/RefundRequestStatusAPI.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  RefundRequestStatusAPI.swift
+//  SwiftAPITester
+//
+//  Created by Madeline Beyl on 10/28/21.
+//
+
+import Foundation
+import RevenueCat
+
```

Comment:
missing `class RefundRequestStatusAPI { ...`

Reply:
was mimicking ErrorCodesAPI, i suppose we want the class for all of them?

Reply:
yeah actually, none of the swift tester files have classes

Reply:
Ah, I see now. Yeah, this is fine then 😄